spec/controllers/grade_entry_forms_controller_spec.rb

```diff
@@ -566,4 +566,58 @@
       include_examples 'switch assignment tests'
     end
   end
+
+  describe '#populate_grades_table' do
+    before(:each) do
+      get_as role, :populate_grades_table, params: { course_id: course.id, id: grade_entry_form_with_data.id }
+    end
+    context 'an instructor' do
+      let(:data_fields) { %w[_id released_to_student user_name first_name last_name hidden section_id] }
+      it 'returns a 200 response' do
+        expect(response.status).to eq 200
+      end
+      it 'returns data' do
+        expect(response.parsed_body['data'].length).to be > 0
+      end
+      it 'returns all student data' do
+        expect(response.parsed_body['data'].length).to eq Student.count
```

Comment:
I'm trying to make sure that tests are more future-proof for when we add tests with multiple courses so can this be `course.students.count`

Reply:
right, duh